server/maintenance_requests.go

```diff
@@ -1,34 +1,27 @@
 // This file provides NewApp() with functions for maintenance requests
 package main
 
-import(
-	"github.com/gin-gonic/gin"
-	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
-	"time"
+import (
+	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
-	"encoding/json"
-)
+	"time"
 
-/*type CreateRequest struct {
-	Request string `json:"request"`
-	UserSubmitted int `json:"id"`
-	DateSubmitted time.Time `json:"date"`
-}*/
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+	"github.com/gin-gonic/gin"
+)
 
 /* AllMaintenanceRequests returns all the maintenance_requests from the database
 *  AllMaintenanceRequests is a function of app */
-func (app *App) AllMaintenanceRequests(c *gin.Context) {
-	/* Get data using psql.AllMaintenanceRequests and pass in app.db for the
-	* connection */
-
-	/* Takes the data returned in the psql func and formats it to a valid format
-	* using the gin.H func. See if it you can just use a Struct */
-
-	/* Use the gin.context.JSON func to return a status and the data. See auth.go
-	* for an example */
+func (app *App) GetMaintenanceRequests(c *gin.Context) {
+	var req = app.db.AllMaintenanceRequests()
+	c.JSON(http.StatusOK, gin.H{"maintenance_requests": req})
+	fmt.Println("maintenance_requests: ", app.db.AllMaintenanceRequests())
 }
 
+// AllMaintenanceRequests returns all the maintenance_requests from the database
```

Comment:
function comments start with the name of the function, and there's no newline separating them. So this comment should be something like `CreateMaintenanceRequest returns all the...`

Reply:
bump; change the function comment to match the function name please!